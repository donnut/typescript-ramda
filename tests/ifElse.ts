import R_ifElse = require('../ramda/dist/src/ifElse');

declare const string_to_boolean: (x: string) => boolean;
declare const string_to_number: (x: string) => number;
declare const string_to_object: (x: string) => object;

// @dts-jest:pass:snap
R_ifElse(string_to_boolean, string_to_number);
// @dts-jest:pass:snap
R_ifElse(string_to_boolean, string_to_number, string_to_object);
// @dts-jest:pass:snap
R_ifElse(string_to_boolean)(string_to_number)(string_to_object);
