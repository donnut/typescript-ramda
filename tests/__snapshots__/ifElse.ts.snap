// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.ifElse(string_to_boolean)(string_to_number)(string_to_object) 1`] = `"(value: string) => number | object"`;

exports[`R.ifElse(string_to_boolean, string_to_number) 1`] = `"<V>(onFalse: (value: string) => V) => (value: string) => number | V"`;

exports[`R.ifElse(string_to_boolean, string_to_number, string_to_object) 1`] = `"(value: string) => number | object"`;
